go.mod

```
@@ -1,24 +1,24 @@
 module github.com/gardener/gardener-extension-os-ubuntu
 
-go 1.13
+go 1.14
 
 require (
-	github.com/gardener/gardener v1.2.1-0.20200409121648-9dd9214f12a1
+	github.com/gardener/gardener v1.6.5
 	github.com/gobuffalo/packr v1.30.1
-	github.com/gobuffalo/packr/v2 v2.5.1
-	github.com/onsi/ginkgo v1.10.1
-	github.com/onsi/gomega v1.7.0
+	github.com/gobuffalo/packr/v2 v2.8.0
+	github.com/onsi/ginkgo v1.11.0
+	github.com/onsi/gomega v1.8.1
 	github.com/spf13/cobra v0.0.6
-	k8s.io/apimachinery v0.17.0
-	sigs.k8s.io/controller-runtime v0.4.0
+	k8s.io/apimachinery v0.17.6
+	sigs.k8s.io/controller-runtime v0.5.4
 )
 
 replace (
-	k8s.io/api => k8s.io/api v0.16.8 // 1.16.8
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.16.8 // 1.16.8
-	k8s.io/apimachinery => k8s.io/apimachinery v0.16.8 // 1.16.8
-	k8s.io/client-go => k8s.io/client-go v0.16.8 // 1.16.8
-	k8s.io/code-generator => k8s.io/code-generator v0.16.8 // 1.16.8
-	k8s.io/component-base => k8s.io/component-base v0.16.8 // 1.16.8
+	k8s.io/api => k8s.io/api v0.17.6 // 1.17.6
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.6 // 1.16.8
```

You could remove the comments at the end, they were kind of useful when we were replacing with only commit sha and human readable tags were not supported..

```suggestion
	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.6
```